tox.ini

```diff
@@ -23,17 +23,22 @@ match-dir = (?!migrations)
 
 [pytest]
 DJANGO_SETTINGS_MODULE = test_settings
-addopts = --cov enterprise --cov integrated_channels --cov-report term-missing --cov-report xml
+addopts = --cov enterprise --cov consent --cov integrated_channels --cov-report term-missing --cov-report xml
 norecursedirs = .* docs requirements
 
 [testenv]
+whitelist_externals =
+    rm
+    touch
 deps =
     ficus: -r{toxinidir}/requirements/test-ficus.txt
     ginkgo: -r{toxinidir}/requirements/test-ginkgo.txt
     hawthorn: -r{toxinidir}/requirements/test-hawthorn.txt
     master: -r{toxinidir}/requirements/test-master.txt
 commands =
+    touch tests/__init__.py
     py.test -Wd {posargs}
+    rm tests/__init__.py
```

Comment:
With the changes to the testing organization, I need to temporarily be able to access the `tests` package in some imports. Note that the same thing is being done below for quality testing. I'm not 100% sure why we don't just keep the `__init__.py`, though.